torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,7 @@
+def replace_all_uses_with(fx_module, old_op, old_target, new_op, new_target):
+    """Modifies all nodes  in nodes which match the op code and target of old_node, and updates them to match new_node"""
+    for node in fx_module.graph.nodes:
+        if node.op == old_op and node.target == old_target:
+            node.op = new_op
+            node.target = new_target
+    fx_module._generate_forward()
```

Comment:
Upon further discussion, this is actually fine because--even if GraphModule instances are copied via `copy()` or `deepcopy()`, the instance still gets a new type via the `__new__` method. I might have a PR in a bit to make this API public and more clear